decidim-core/app/events/decidim/report_created_event.rb

```diff
@@ -0,0 +1,28 @@
+# frozen-string_literal: true
+
+module Decidim
+  class ReportCreatedEvent < Decidim::Events::SimpleEvent
```

Comment:
You probably want to include [AuthorEvent](https://github.com/decidim/decidim/blob/develop/decidim-core/lib/decidim/events/author_event.rb)

Reply:
Hm, I'm not sure about this one, because at the moment we're not really using any info from the author in `ReportCreatedEvent`.